docs/changelog.rst

```diff
@@ -1,6 +1,21 @@
 Changes in jupyter-core
 =======================
 
+4.1
+---
+
+4.1.0
+~~~~~
+
+`on
+GitHub <https://github.com/jupyter/jupyter_core/releases/tag/4.1>`__
+
+- Add `jupyter.py` module, so that :command:`python -m jupyter` always works.
```

Comment:
This repo doesn't have the single-backticks-is-double-backticks thing enabled. If you want it, the setting is here: https://github.com/jupyter/jupyter_core/blob/7d71e36d62a52e2c55593a092177447ff53b3995/docs/conf.py#L87